### ISSUE TITLE
Migrate data-format-description.ts to separate package to allow use in the card-creator too

### ISSUE THREAD

**Issue by Greenheart** [Greenheart/swipeforfuture.com#59]
If this would be moved to a separate package, it could be imported with npm directly from a git repo. This would allow us to update the card types on a single place, making maintenance easier.

**Comment by Greenheart** [comment]
With TypeScript added to the game, and `ContentTypes.ts` within the game itself, this is no longer needed.

After #100 is implemented, the card creator will be a part of the game itself too
